src/main/kotlin/me/zeroeightsix/kami/gui/InteractableScreen.kt

```diff
@@ -0,0 +1,50 @@
+package me.zeroeightsix.kami.gui
+
+import imgui.ConfigFlag
+import imgui.ImGui
+import imgui.impl.glfw.ImplGlfw
+import imgui.wo
+import net.minecraft.client.gui.screen.Screen
+import net.minecraft.text.Text
+import org.lwjgl.glfw.GLFW
+
+/**
+ * class for a screen for which input should be handled.
+ * includes callback handlers
+ */
+abstract class InteractableScreen(title: Text) : Screen(title) {
```

Comment:
This name seems to imply a more generic "interaction" class, but it actually implements dear imgui (something specific).
The name should reflect that - e.g. `ImGuiScreen`